view/theme/vier/style.css

```diff
@@ -456,7 +456,7 @@ pre code {
 	font-weight: bold;
 }
 
-#group-widget-showmore, #sidebar-new-circle, #sidebar-edit-circles, #sidebar-new-group, #group-widget-collapse, #group-list-rsidebar-right, #sidebar-uncircled,
+#group-widget-showmore, #sidebar-new-circle, #sidebar-edit-circles, #sidebar-new-group, #group-widget-collapse, #grouplist-rsidebar-right, #sidebar-uncircled,
```

Comment:
Please remove this change that is a revert of a previous PR's change request.

Reply:
Ok so hypothetically this should be rebased to 8ec3f17.  Let me know if it worked.

Reply:
Tried to push after selecting your fork and got this response.

Pushing to https://github.com/anubis2814/friendica
To https://github.com/anubis2814/friendica
 ! [rejected]            develop -> develop (non-fast-forward)
error: failed to push some refs to 'https://github.com/anubis2814/friendica'

Reply:
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
Like I said, you need to force push after a rebase. You need to check the box to "Force push".

Reply:
Ok it looks like it works finally. Thanks for helping with my confusion

Reply:
Now you just need to amend the lone commit to reflect your changes. You can do this with the "Amend this commit" checkbox in Fork, change the commit message above and commit again. You will have to force push once more.

Reply:
Where can I find this lone commit?

Reply:
Forks says I have no uncommited changes

Reply:
It's pretty hard for me to use git asynchronously through github message. I'll fix your branch this time around, hopefully next time will be smoother, provided you remember to pull the latest `develop` first 😉

Reply:
Thanks, I've been recommended to play a game called "oh my git" that will hopefully help me understand it better.